addons/l10n_cl/models/account_move_line.py

```diff
@@ -9,9 +10,14 @@ class AccountMoveLine(models.Model):
 
     def _l10n_cl_prices_and_taxes(self):
         self.ensure_one()
+        if self.display_type != 'product':
+            return {
+                'line_amounts': {'line_description': self.name},
+                'price_subtotal': 0,
+            }
         invoice = self.move_id
         included_taxes = self.tax_ids.filtered(lambda x: x.l10n_cl_sii_code == 14) if self.move_id._l10n_cl_include_sii() else self.tax_ids
-        if not included_taxes:
+        if included_taxes:
```

Comment:
Why do we change the logic if a bit further we override the logic by the _l10n_cl_get_line_amounts?  Maybe we should just leave this method alone and indicate it as "deprecated, to be removed in master".  
Maybe it was done here to have the line_description, but then we could add it in the _get_line_amounts as well as we need to add the line_description to the qweb anyways.

Reply:
@jco-odoo This solution impacts 3 versions:
16.0 -> Leave at it is? (we talked that in 16 we kept it)
17.0 -> Define as deprecated the pdf methods. (?)
master -> remove pdf method completely (?)